src/style.css

```
@@ -1 +1,10 @@
 /* styles go here */
+
+.mainFieldset {
```

`fieldset` will be enough
```suggestion
.fieldset {
```